website/routes.py

```diff
@@ -1166,6 +1166,16 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/contributor/remove/',
+                '/project/<pid>/node/<nid>/contributor/remove/',
```

Comment:
Not a huge fan of ambiguous URLs, but I guess that's not new on this PR. You seem to be following an established pattern here.